packages/build/tests/install/tests.js

```diff
@@ -63,13 +63,13 @@ test('Functions: does not install dependencies unless opting in', async (t) => {
 })
 
 test('Functions: does not install dependencies unless opting in (with esbuild)', async (t) => {
-  await runInstallFixture(t, 'optional', [], { flags: { featureFlags: 'buildbot_esbuild' } })
-  t.false(await pathExists(`${FIXTURES_DIR}/optional/functions/node_modules/`))
+  await runInstallFixture(t, 'optional-esbuild', [])
```

Comment:
We couldn't keep reusing the same fixture for both bundlers, because the bundler selection now comes from the configuration file. I have copied the `optional` and `optional-many` fixtures into `optional-esbuild` and `optional-many-esbuild` for this reason.